pkg/netutil/isolate_linux.go

```diff
@@ -40,3 +40,24 @@ func RecoverPort(port int) error {
 	_, err := exec.Command("/bin/sh", "-c", cmdStr).Output()
 	return err
 }
+
+// AddLatency adds latency in millisecond scale with +/-10ms random variations.
+func AddLatency(ms int) error {
```

Comment:
we should be able to set variations. say i just want 10ms latency. then 10ms variations is 100%. it does not always make sense.